.github/CONTRIBUTING.md

```diff
@@ -207,24 +208,30 @@ if the original datum has been deleted - BYOND recycles the references.
 
 UID's are actually unique; they work off of a global counter and are not recycled. Each datum has one assigned to it when it's created, which can be
 accessed by `[datum.UID()]`. You can use this as a snap-in replacement for `\ref` by changing any `locate(ref)` calls in your code to `locateUID(ref)`.
-Usage of this system is mandatory for any /Topic( calls, and will produce errors in Dream Daemon if it's not used. `<a href='?src=[UID()];'>`, not `<a href='?src=\ref[src];'`.
+Usage of this system is mandatory for any `Topic()` calls, and will produce errors in Dream Daemon if it's not used.
+```DM
+//Bad
+"<a href='?src=\ref[src];'>Link!</a>"
 
-### Use var/name format when declaring variables
+//Good
+"<a href='?src=[UID()];'>Link!</a>"
+```
+
+### Use `var/name` format when declaring variables
 While DM allows other ways of declaring variables, this one should be used for consistency.
 
 ### Tabs, not spaces
 You must use tabs to indent your code, NOT SPACES.
 
-(You may use spaces to align something, but you should tab to the block level first, then add the remaining spaces)
+(You may use spaces to align something, but you should tab to the block level first, then add the remaining spaces.)
 
 ### No hacky code
-Hacky code, such as adding specific checks (ex: `istype(src, /obj/whatever)`), is highly discouraged and only allowed when there is ***no*** other option. (
-Protip: 'I couldn't immediately think of a proper way so thus there must be no other option' is not gonna cut it here! If you can't think of anything else, say that outright and admit that you need help with it. Maintainers exist for exactly that reason.)
+Hacky code, such as adding specific checks (ex: `istype(src, /obj/whatever)`), is highly discouraged and only allowed when there is ***no*** other option. (Protip: 'I couldn't immediately think of a proper way so thus there must be no other option' is not gonna cut it here! If you can't think of anything else, say that outright and admit that you need help with it. Maintainers exist for exactly that reason.)
```

Comment:
```suggestion
Hacky code, such as adding specific checks (ex: `istype(src, /obj/whatever)`), is highly discouraged and only allowed when there is ***no*** other option. (Protip: 'I couldn't immediately think of a proper way so thus there must be no other option' is not gonna cut it here! If you can't think of anything else, say that outright and admit that you need help with it. Maintainers and other contributors who can help you exist for exactly that reason.)
```

Reply:
Could mention PR Reviewers as well? Or just leave it as "Contributors".